src/commands/appdistribution-testers-remove.ts

```diff
@@ -0,0 +1,33 @@
+import {Command} from "../command";
+import * as utils from "../utils";
+import {requireAuth} from "../requireAuth";
+import {FirebaseError} from "../error";
+import {needProjectNumber} from '../projectUtils';
+import {AppDistributionClient} from '../appdistribution/client';
+import {getEmails} from '../testerEmailParser';
+
+
+module.exports = new Command("appdistribution:testers:remove [emails...]")
+    .description("Delete Testers")
+    .option(
+        "--file <file>",
+        "a path to a file containing a comma separated list of tester emails to be removed"
+    )
+    .before(requireAuth)
+    .action(async (emails: string[], options?: any) => {
+      const projectNumber = await needProjectNumber(options);
+      const request = new AppDistributionClient();
+      const emailsArr = getEmails(emails, options.file);
+      let deleteResponse;
+      try {
+        console.log("Deleting the following testers: " + emailsArr + " from project " + projectNumber);
```

Comment:
`emailsArr` could be a very long array. Consider just printing the count i.e. `Deleting X testers from project Y`

Reply:
+1, and do the same in the other command